app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,90 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    @task = Task.find_by(id: params[:id])
+
+    if @task.nil?
+      # head :not_found
+      redirect_to tasks_path
+      return
+    end
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_params)
+    if @task.save
+      redirect_to task_path(@task.id)
+      return
+    else
+      render :new, :bad_request
```

Comment:
Good job specifying the status.  😃

Reply:
Thank you so much Kaida!  It was a huge encouragement! ❤️🧡💛💚💙💜🖤